packages/transformers/xml/src/atom.js

```diff
@@ -34,11 +34,10 @@ export function content(
       break;
     case 'xhtml': {
       let fragment = element.ownerDocument.createDocumentFragment();
-      let child = element.firstChild;
-      while (child) {
+      let child;
+      while ((child = element.firstChild)) {
         element.removeChild(child);
         fragment.appendChild(child.cloneNode(true));
-        child = child.nextSibling;
```

Comment:
so it only goes down the tree through firstChild instead of through all of the siblings of the element now?

Reply:
Previously, it removed all children of `element`.

Now, it repeatedly removes `element.firstChild` until that is null (= no more children, we are done).